docs/en/guides/segment-template.json

```diff
@@ -0,0 +1,144 @@
+        "type": "randomString",
+        "length": "10",
+        "letters": true,
+        "numbers": true
+    },
+    "endpointName": {
+        "type": "randomString",
+        "length": "10",
+        "prefix": "test_",
+        "letters": true,
+        "numbers": true,
+        "domainSize": 10
+    },
+    "startTime": {
+        "type": "sequence",
+        "min": "1650510807000",
+        "max": "1650512807000"
+    },
+    "latency": {
+        "type": "randomInt",
+        "min": 100,
+        "max": 1000
+    },
+    "error": {
+        "type": "fixedInt",
+        "value": "0"
+    },
+    "tags": {
+        "type": "randomList",
+        "size": 5,
+        "item": {
+            "key": {
+                "type": "randomString",
+                "length": "10",
+                "prefix": "test_tag_",
+                "letters": true,
+                "numbers": true,
+                "domainSize": 10
+            },
+            "value": {
+                "type": "randomString",
+                "length": "10",
+                "prefix": "test_value_",
+                "letters": true,
+                "numbers": true
+            }
+        }
+    },
+    "spans": {
+        "type": "randomList",
+        "size": 5,
+        "item": {
+            "spanId": {
+                "type": "sequence",
+                "min": "0",
+                "max": "5"
+            },
+            "parentSpanId": {
+                "type": "sequence",
+                "min": "-1",
+                "max": "4"
+            },
+            "startTime": {
+                "type": "randomInt",
+                "min": "1650510807000",
+                "max": "1650512807000",
+                "domainSize": 10
+            },
+            "endTime": {
+                "type": "randomInt",
+                "min": "1650510807000",
+                "max": "1650512807000",
+                "domainSize": 10
+            },
+            "operationName": {
+                "type": "randomString",
+                "length": "10",
+                "prefix": "test_endpoint_",
+                "letters": true,
+                "numbers": true
+            },
+            "peer": {
+                "type": "randomString",
+                "length": "10",
+                "prefix": "test_peer_",
+                "letters": true,
+                "numbers": true
+            },
+            "spanLayer": {
+                "type": "randomInt",
+                "min": "0",
+                "max": "4"
+            },
+            "componentId": {
+                "type": "randomInt",
+                "min": "0",
+                "max": "4"
+            },
+            "error": {
+                "type": "fixedBool",
+                "value": "false"
+            },
+            "tags": {
+                "type": "randomList",
+                "size": 5,
+                "item": {
+                    "key": {
+                        "type": "randomString",
```

Comment:
Could we have `stringCollection[str1, str2, str3]`? Don't have to be in this PR, but would be a good feature for testing.